src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -998,6 +998,254 @@ public with sharing class ADDR_Addresses_TEST {
 
     /*********************************************************************************************************
     @description
+        Update Contact's mailing address (with State & Country Picklists enabled) and mark it as override,
+        to cause creation of a new non-default Address record with the State & Country values set as expected.
+    verify:
+        Contact gets new address
+        HH sticks with default Address
+        New Address created with State value set as expected
+    **********************************************************************************************************/
+    static testMethod void testContactAddressOverrideForStateChangeWithStateAndCountryPicklistEnabled() {
```

Comment:
Suggestion: `testAddressOverrideCreatesNewAddressWhenStateIsChanged`?

Reply:
I like that. Possibly `testAddressOverrideCreatesNewAddressWhenStateCodeIsChanged` ? That way, it speaks to State and Country picklists being enabled. Thoughts?

Reply:
The State/Country picklist being enabled might be part of the test description and it is clear from the first statement, the test is not executed otherwise. It is fine as long as the name is not too long so readability is lost, or too short so it is not clear what is being tested and expected.